.vitepress/config.mts

```diff
@@ -35,6 +35,339 @@ function nav(): DefaultTheme.NavItem[] {
+            },
+            {
+              text: 'Update Executor',
+            }
+          ],
+        }
+      ],
+    },
+    {
+      text: 'Accounts',
+      collapsed: true,
+      items: [
+        {
+          text: 'Register Accounts',
+        },
+        {
+          text: 'Unregister Accounts',
+        },
+        {
+          text: 'Update Account Metadata'
+        },
+        // ...?
+      ],
+    },
+    {
+      text: 'Assets',
+      collapsed: true,
+      items: [
+        {
+          text: 'Assets', // All Assets?
+          collapsed: true,
+          items: [
+            {
+              text: 'Register Assets',
+            },
+            {
+              text: 'Unregister Assets',
+            },
+            {
+              text: 'Transfer Assets Between Accounts',
+            },
+            // ...?
+          ],
+        },
+        {
+          text: 'Numeric Assets',
+          collapsed: true,
+          items: [
+            {
+              text: 'Work with Numeric Assets',
+            }
+          ],
+        },
+        {
+          text: 'Store Assets',
+          collapsed: true,
+          items: [
+            {
+              text: 'Work with Store Assets',
+            }
+          ],
+        },
+        {
+          text: 'Mintable Assets',
+          collapsed: true,
+          items: [
+            {
+              text: 'Mint Assets',
+            },
+            {
+              text: 'Mint More of a Mintable Asset',
+            },
+            {
+              text: 'Burn Assets',
+            },
+          ],
+        },
+        {
+          text: 'Non-Mintable Assets',
+          collapsed: true,
+          items: [
+            {
+              text: 'Work with Non-Mintable Assets',
+            },
+          ],
+        },
+        {
+          text: 'Tokens',
+          collapsed: true,
+          items: [
+            {
+              text: 'Create Asset-backed Tokens',
+            },
+            {
+              text: 'Create Non-Fungible Tokens (NFTs)',
+            },
+          ],
+        },
+      ],
+    },
+    {
+      text: 'Domains',
+      collapsed: true,
+      items: [
+        {
+          text: 'Register Domains',
+        },
+        {
+          text: 'Unregister Domains'
+        },
+        {
+          text: 'Update Domain Metadata',
+        },
+        // ... 
+        // access management for domains?
+      ],
+    },
+    {
+      text: 'Events and Filters',
+      collapsed: true,
+      items: [
+        {
+          text: 'Pipeline Events',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Data Events',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Trigger Events',
+          collapsed: true,
+          items: [],
+        },
+        {
+          text: 'Advanced Filtering',
+          collapsed: true,
+          items: [],
+        },
```

Comment:
this also needs more details, what kind of example can we include here? just "work with {event type}"? or something else?